src/server/api/routes/channels-message-router.js

```diff
@@ -4,9 +4,32 @@ const router = express.Router({ mergeParams: true });
 
 const channelsMessageController = require('../controllers/channels-message-controller');
 
-router.get('/:userId', (req, res, next) => {
+/**
+ * @swagger
+ * /channels-message/last-messages:
+ *  get:
+ *    summary: Get channels last messages by user id
+ *    description:
+ *      Will return users last conversation with a matching user ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: query
+ *       name: userId
+ *       schema:
+ *         type: string
+ *         required: true
+ *         description: The id of the user to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+
+router.get('/last-messages', (req, res, next) => {
   channelsMessageController
-    .getChannelsId(req.params.userId)
+    .getChannelsId(req.query.userId)
```

Comment:
Why did you switch from using param to query here? Just curious.

Reply:
It is my preference :) I think here better use query.